src/__tests__/shared/components/network-status/NetworkStatusMessage.test.tsx

```diff
@@ -1,27 +1,30 @@
 import { render } from '@testing-library/react'
-import { renderHook } from '@testing-library/react-hooks'
 import React from 'react'
 
 import { useTranslation } from '../../../../__mocks__/react-i18next'
 import { NetworkStatusMessage } from '../../../../shared/components/network-status'
 import { useNetworkStatus } from '../../../../shared/components/network-status/useNetworkStatus'
 
 jest.mock('../../../../shared/components/network-status/useNetworkStatus')
-const useNetworkStatusMock = (useNetworkStatus as unknown) as jest.MockInstance<
-  ReturnType<typeof useNetworkStatus>,
-  any
->
 
-const englishTranslationsMock = {
-  'networkStatus.offline': 'you are working in offline mode',
-  'networkStatus.online': 'you are back online',
-}
+describe('NetworkStatusMessage', () => {
+  const useNetworkStatusMock = (useNetworkStatus as unknown) as jest.MockInstance<
+    ReturnType<typeof useNetworkStatus>,
+    any
+  >
 
-const { result } = renderHook(() => useTranslation() as any)
-result.current.t = (key: keyof typeof englishTranslationsMock) => englishTranslationsMock[key]
-const { t } = result.current
```

Comment:
This whole bit was really strange.   It took me ages to realise that `useTranslation` is being mocked and not actually a hook any more so actually rendering it it pointless and confusing.